lib/sles4sap/ipaddr2.pm

```diff
@@ -358,6 +371,50 @@ sub ipaddr2_deployment_sanity {
         $count = grep(/running/, @$res);
         die "VM $vm is not fully running" unless $count eq 2;    # 2 is two occurrence of the word 'running' for one VM
     }
+}
+
+=head2 ipaddr2_os_connectivity_sanity
+
+    ipaddr2_os_connectivity_sanity()
+
+Run some OS level checks about internal connectivity.
+die in case of failure
+
+- bastion has to be able to ping the internal VM using the internal private IP
+- bastion has to be able to ping the internal VM using the internal VM hostname
+
+=over 1
+
+=item B<bastion_ip> - Public IP address of the bastion. Calculated if not provided.
+                      managed as argument not to have to call ipaddr2_bastion_pubip many time,
+                      so not to have to query az each time
+
+=back
+=cut
+
+sub ipaddr2_os_connectivity_sanity {
+    my (%args) = @_;
+    $args{bastion_ip} //= ipaddr2_bastion_pubip();
+
+    # proceed_on_failure needed as ping or nc
+    # coul dbe missing on the qcow2 running these commands
```

Comment:
Please look at this http://openqaworker15.qa.suse.cz/tests/287112#step/configure/18